test/src/models/form_group_test.dart

```diff
@@ -122,7 +122,7 @@ void main() {
     test('Set value to FormGroup', () {
       final form = FormGroup({
         'name': FormControl(),
-        'email': FormControl(),
+        'email': FormControl<String>(),
```

Comment:
This will not allow creating a control with dynamic type as:
```dart
final control = FormControl(
   value: "I'm a String control",
   validators: [Validators.email], // exception is thrown
);
```
So I suggest leaving the Email validator as **dynamic**